mods/ra/maps/allies-02/rules.yaml

```diff
@@ -22,7 +22,7 @@ World:
 			normal: Normal
 			hard: Hard
 			tough: Real tough guy
-		Default: easy
+		Default: normal
```

Comment:
IIRC difficulties are then displayed out of order in the dropdown (you know, normal -> easy -> hard).
I don't particularly mind, but that might have been the reason this wasn't done before.

Reply:
Double checked it. Easy is still on top on the dropdown.

Reply:
Huh... tested some more and it seems to be a TD-exclusive issue, and actually not related to what's set as default.

Anyway, no impact on this PR then.